### PR TITLE
QC checks and tagging system for Mondo

### PR DESCRIPTION
**Pull request by matentzn**
SPARQL check for two gene based diseases in a subsumption hierarchy

### CODE REVIEW

File: src/ontology/README-editors.md [hunk 1]
```diff
@@ -101,3 +101,35 @@ Check the build status here: [![Build Status](https://travis-ci.org/monarch-init
...
+# Mondo QC report and metrics
+
+Mondo QC is split into two conceptual parts:
+
+1. Violations
+2. Warnings
+
+Violations _have to be fixed_ and are executed every time a pull request is created. They are implemented as sparql queries with a postfix "violation", like `no-label-violation.sparql`. To create a new violation, we create a new SPARQL SELECT query that queries for the violation, and name it including the `-violation` postfix. The violation query is executed using ROBOT verify, which will make the CI (Travis/Jenkins) pipeline fail if the query has more than `0` results.
+
+_Warnings_ do not have to be fixed, but are indications of something going wrong. Warnings are implemented in the same way as `violation` queries with a SPARQL Select query, named with a postfix (example: `single-child-warning.sparql`).
```
Comment (dhimmel) [inline src/ontology/README-editors.md:113]: very helpful docs!

Comment (matentzn) [reply src/ontology/README-editors.md:113]: Thank you! :) A bit of a surprising comment in a buried PR.. What is your interest in this?

**Comment by matentzn** [comment]
There are too many changes in the PR to expect you to review this @cmungall but I want to move on, so I reviewed it myself again, made some changes and have it ready to merge. You can look it over if you want, but this PR mainly has:

a bunch of new tests (which will again be changed, because I found a much cooler way to implement them using ROBOT report), the pattern and tagging system (components/mondo-tags.owl).

File: src/ontology/README-editors.md (hunk 1 above)
Comment (dhimmel) [reply src/ontology/README-editors.md:113]: Well I tend to look through pull requests to learn about projects. In this case, I was especially interested in the SPARQL violation checks as a way to enforce a higher level of data standardization and cleanliness in ontologies.

For example, It looks like the [`two-label-violation.sparql`](https://github.com/monarch-initiative/mondo/blob/c872b04bf8fed765b579dbfb95f07b6f0358c28f/src/sparql/two-label-violation.sparql) is a good way to make sure issues like https://github.com/EBISPOT/efo/issues/871 don't arise.

Comment (matentzn) [reply src/ontology/README-editors.md:113]: Wow cool! :P I have seen your name pop up here and there on github, so I thought I'd ask. Noble to learn this way! :) I will resolve this comment now, see you in another thread somewhere else :P

**Pull request merged**